packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -536,6 +543,133 @@ export function getPreviewMiniflareOptions(
 		unstable_readConfig({ config: configPath })
 	);
 
+	const entryWorkerConfig = workerConfigs[0];
+
+	const assetWorkers: Array<WorkerOptions> =
```

Comment:
I am really, _really_ not a fan of how this works in Vite.

Reply:
Without it, `vite preview` was just always delegating 404s to the user worker, but I've now realized that this is just because of the same issue we saw in dev! I can remove all this now :)